src/app/components/FilterPopup/FilterPopup.jsx

```diff
@@ -292,12 +298,13 @@ class FilterPopup extends React.Component {
         Cancel
       </button>
     );
-    const resetButton = (
+    const resetButton = (ref = '') => (
```

Comment:
Are ref attributes set to empty string's as a default? Just wondering.

Reply:
This is syntax not specific to ref but to any function parameter/argument that is passed that you want to set a default value for. In this case, the parameter variable is `ref` and if no value gets passed to the function, the default value is an empty string `''`.

Reply:
This is also new to Javascript.